### DISCUSSION TITLE
Entity to leverage cheap Octopus Intelligent Go prices

### DISCUSSION THREAD

**Discussion by kaihendry** [BottlecapDave/HomeAssistant-OctopusEnergy#1024] [Q&A] [answered]
I have my dehumidifer attached to a plug with an automation like so:

        - id: turn_on_e724_when_off_peak
          alias: Turn On TP-Link Smart Plug E724 When Off-Peak
          trigger:
            - platform: state
              entity_id: binary_sensor.octopus_energy_electricity_redacted_off_peak
              to: 'on'  # Off-peak starts
          action:
            - service: switch.turn_on
              entity_id: switch.tp_link_smart_plug_e724
        
        - id: turn_off_e724_when_not_off_peak
          alias: Turn Off TP-Link Smart Plug E724 When Not Off-Peak
          trigger:
            - platform: state
              entity_id: binary_sensor.octopus_energy_electricity_redacted_off_peak
              to: 'off'  # Off-peak ends
          action:
            - service: switch.turn_off
              entity_id: switch.tp_link_smart_plug_e724

I have checked the [FAQ](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/faq/) and I'm not sure I am capturing all the cheap times. I have heard of an entity called `..intelligent_dispatching`, is there some entity that somehow encapsulates all the times electricity is cheap?

**Comment by BottlecapDave** [comment]
You're correct that if you're an intelligent user, there is a sensor that is better suited (https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/entities/intelligent/#is-dispatching). Please note the warning as the pending dispatch reported by OE doesn't necessarily mean you'll get the cheap rate if your vehicle isn't charging at the time. This information doesn't seem to be reported by OE and the dispatch/settings is limited in how often the underlying data refreshes.

**Reply by kaihendry** [reply]
Ah, thanks, though does "intelligent_dispatching" capture that offpeak time too? This is the thing that confused me. Or rather how to write that automation with two entities in mind.

Tbh I don't mind taking the risk that I might get charged extra if my car is not plugged in. Thank you!

**Reply by BottlecapDave** [reply, marked as answer]
As per the docs
> This sensor is used to determine if you're currently in a planned dispatch period (i.e. "smart-charge" determined by Octopus Energy) or are within the standard off peak period

So you only need this sensor.
